### ISSUE TITLE
Drop support for python 2

### ISSUE THREAD

**Issue by inirudebwoy** [inirudebwoy/bamp#22]

**Comment by flyte** [comment]
You can use poetry to publish new versions instead of twine:
https://python-poetry.org/docs/cli/#publish

**Comment by inirudebwoy** [comment]
Yes, omg thanks :) I already removed twine and setup GitLab CI to use poetry. Thanks again.
This is a new version https://github.com/inirudebwoy/bamp/releases/tag/v1.0.1
